### ISSUE TITLE
cta-section: additional spacing problem

### ISSUE THREAD

**Issue by skycryer** [carbon-design-system/carbon-web-components#1014]
### Description

The cta setion has extra spacing and fixed height for the items. With this spacing there is some more spacing then it should
<img width="1204" alt="Screenshot 2022-07-22 at 14 34 21" src="https://user-images.githubusercontent.com/3725699/180441324-91214335-eb20-47df-8851-5a0a4eef2a2f.png">
Fixed item height

<img width="1204" alt="Screenshot 2022-07-22 at 14 34 39" src="https://user-images.githubusercontent.com/3725699/180441355-7b9590dd-ee95-4fe3-8139-55755d87aa31.png">
Extra spacing below component.

### Component(s) impacted

cta-section

### Carbon Web Components version

1.20.0

### Severity

Severity 1 = The design is broken in a critical way that blocks users from completing tasks or damages the brand. Affects major functionality, no workaround.

### Application/website

IBM CMS

### CodeSandbox example

https://prod-author.roks.cms.cis.ibm.net/content/adobe-cms/language-masters/en/products/watson-orchestrate.html?wcmmode=disabled

### Steps to reproduce the issue (if applicable)

Just add this component over another component. THen you see the component has to much empty spacing below

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-web-components/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-web-components/issues) for duplicate issues

**Comment by proeung** [comment]
@oliviaflory @kennylam This issue looks to be more appropriate on the C4IBM repo side. Would it be okay, if I move it there or are there a way to see issues from this `carbon-web-components` repo on the `carbon-for-ibm-dotcom` side as well?

**Comment by oliviaflory** [comment]
@proeung good catch, I can transfer the issue to the `carbon-for-ibm-dotcom` repo. We are able to see all issues from all our repos in our Zenhub board for future reference 👍
